File: 05_menuOperaciones/Cpp/menuOperEI.cpp
#include<iostream>
using namespace std;
float num1,num2,res;
char op;
int main()
{
    cout<<"Elija una de las siguientes opciones:\n\n\ta) Suma.\n\tb) Resta.\n\tc) Multiplicacion.\n\td) Division.\n\n\t\t\t\t Opcion: ";
    cin>>op;
    if(op=='a')
    {
        cout<<"Escribe primer sumando: ";
        cin>>num1;
        cout<<"Escribe segundo sumando: ";
        cin>>num2;
        res=num1+num2;
        cout<<"La suma es: "<<res<<endl;
    }
    else if(op=='b')
    {
        cout<<"Escribe el minuendo: ";
        cin>>num1;
        cout<<"Escribe el sustraendo: ";
        cin>>num2;
        res=num1-num2;
        cout<<"La resta es: "<<res<<endl;
    }
    else if(op=='c')
    {
        cout<<"Escribe primer factor: ";
        cin>>num1;
        cout<<"Escribe segundo factor: ";
        cin>>num2;
        res=num1*num2;
        cout<<"La suma es: "<<res<<endl;
    }
    else if(op=='d')
    {
        cout<<"Escribe el dividendo: ";
        cin>>num1;
        cout<<"Escribe el divisor: ";
        cin>>num2;
        res=num1/num2;
        cout<<"La suma es: "<<res<<endl;
    }
    else
        cout<<"Esa no es una opcion.\n";
    return 0;
}

File: 05_menuOperaciones/Python/menuOperEI.py
op=input("Elija una de las siguientes opciones:\n\n\ta) Suma.\n\tb) Resta.\n\tc) Multiplicacion.\n\td) Division.\n\n\t\t\t\t Opcion: ")
if op=='a':
    num1=float(input("Ingresa primer sumando: "))
    num2=float(input("Ingresa segundo sumando: "))
    res=num1+num2
    print("La suma es: %f" %res)
elif op=='b':
    num1=float(input("Ingresa el minuendo: "))
    num2=float(input("Ingresa el sustraendo: "))
    res=num1-num2
    print("La resta es: %f" %res)
elif op=='c':
    num1=float(input("Ingresa primer factor: "))
    num2=float(input("Ingresa segundo factor: "))
    res=num1*num2
    print("El producto es: %f" %res)
elif op=='d':
    num1=float(input("Ingresa dividendo: "))
    num2=float(input("Ingresa divisor: "))
    res=num1/num2
    print("La division es: %f" %res)
else:
    print("Esa no es una opcion.")